### PR TITLE
Fixed inline icon alignment issues

### LINKED ISSUE #249: Inventory register more systems styling is off
The plus icon that precedes the register more systems link is not aligned with the text.

screen shot 2017-10-03 at 3 29 59 pm

### PR DESCRIPTION
**Pull request by ryelo**
Fixes #249

### CODE REVIEW

**Comment by Kinlaw** [comment]
@ryelo Two things:

1) I feel like the + icon on Register More should be vertically centered with the text

screen shot 2017-10-06 at 2 44 20 pm

2) The edit and delete icons don't quite look aligned here

screen shot 2017-10-06 at 2 43 44 pm

**Comment by ryelo** [comment]
@Kinlaw I'll check it out and get a fix for it!

**Comment by jharting** [comment]
@ryelo I suspect this entire problem was caused by a css change that affected icons all over the place. I think we should revert/revisit that change instead of fixing the individual places that ended up being affected. Otherise, we'll likely miss some spots. For example, I just found another icon with weird alignment:

screenshot_20171009_171347

**Comment by ryelo** [comment]
@jharting Although that might be 'easier', the reason is because of inconsistent code. So although the 'wrong' way did work before, whatever styles were removed broke them. 

We could definitely investigate to why they are broken now, but IMO it would be better to clean up the code, even if that means combing through files.

**Comment by jharting** [comment]
@ryelo @mattnolting FYI I bisected this and it's caused by https://github.com/RedHatInsights/insights-frontend/commit/86673ce479038071395edb0ea489af73df7efc45#diff-6ed0c4f60620134411aa1d4e72d970c9

i.e. nothing was removed. Instead, it looks like a change intended to have local effect (risk of change icons) actually affected many more spots with icons.

I am fine with whatever approach to fixing this you think is best as long as it addresses all the icons broken by the above.

**Comment by Kinlaw** [comment]
@ryelo this looks good to me. The Webhooks icons still look slightly off but when I inspect them it looks like it's an inconsistency with the icons themselves and not the alignment on the page. It still looks better than what is in insights beta currently so I'm fine with merging this.

@jharting your thoughts?

**Comment by ryelo** [comment]
@Kinlaw I think @mattnolting might be working to improve some stuff in this branch too.

**Comment by mattnolting** [comment]
I'm working on this, hopefully will push today

**Comment by Kinlaw** [comment]
Another less-commonly-known use case is the error box in the front end. The exclamation mark is off. To reproduce:

1) comment out line 157 in states -> digests -> digests.controller.js
`uri.query(AccountService.current());`
2) Go to Executive Report
3) Switch the Account Number to 6
4) Click Download PDF.

screen shot 2017-10-11 at 10 29 10 am

**Comment by mattnolting** [comment]
I'm looking at all icons-inlines

**Comment by Kinlaw** [comment]
Before we merge this, can we squash this down to one or two commits?

**Comment by jharting** [comment]
@ryelo @Kinlaw @mattnolting FYI looks like the icon alignment problem leaked into stable. Once this is merged we should backport to stable also.

File: app/js/components/maintenance/maintenanceTable/maintenanceTableActions.jade [hunk 1]
```diff
@@ -20,10 +20,9 @@
         ng-mousedown="checkboxRowClick($event, action.id)")
         td.text-center(ng-show='edit')
           input(ng-model='checkboxes.items[action.id]', type="checkbox")
-        td.overflow
-          span.icon-inline(ng-if='action._type === 2')
-            i.fa.fa-exclamation-circle(tooltip="{{'This action is already part of another plan' | translate}}")
-          a.action(ng-click='table.showSystemModal(table.params.item, action.rule)') {{action.display}}
+        td.overflow(ng-class='{icon-inline: action._type === 2}')
```
Comment (Kinlaw) [inline app/js/components/maintenance/maintenanceTable/maintenanceTableActions.jade]: icon-inline needs to be in quotes. Chrome browser console is reporting a syntax error

File: app/js/components/overview/widgets/maintenancePlanLite/maintenancePlanLite.jade [hunk 2]
```diff
@@ -5,13 +5,14 @@ section.widget-block
     span.light(translate) Summary
     a.cta-link(ui-sref='app.maintenance', translate) View planner
   .maintenance-plan-lite
-    .row.row-short.padding-top(ng-if='getFirstSuggestion()')
+    //- .row.row-short.padding-top(ng-if='getFirstSuggestion()')
+    .row.row-short.padding-top
```
Comment (Kinlaw) [inline app/js/components/overview/widgets/maintenancePlanLite/maintenancePlanLite.jade]: ng-if='getFirstSuggestion()' should be here. Account 540155 shows a 0 action suggestion.

**Pull request merged**